content/attachments/instance-0333c621-a8e0-4bd5-86b9-d362e24f6161/instance-0333c621-a8e0-4bd5-86b9-d362e24f6161-příloha.ttl

```diff
@@ -168,9 +168,9 @@
   <http://onto.fel.cvut.cz/ontologies/application/ontoGrapher/scheme> <https://slovník.gov.cz/legislativní/sbírka/359/2011/glosář>;
   <http://onto.fel.cvut.cz/ontologies/application/ontoGrapher/vocabulary> <https://slovník.gov.cz/legislativní/sbírka/359/2011> .
 
-<https://slovník.gov.cz/datový/pracovní-prostor/pojem/příloha/instance-0333c621-a8e0-4bd5-86b9-d362e24f6161>
+<http://onto.fel.cvut.cz/ontologies/slovník/agendový/popis-dat/pojem/příloha/instance-0333c621-a8e0-4bd5-86b9-d362e24f6161>
```

Comment:
a všechny ostatná změny - obávám se, že mergnutím bychom způsobili tuto chybu - https://github.com/opendata-mvcr/ssp/runs/6899263868?check_suite_focus=true. První tedy asi musíme vyřešit opravu tohoto problému

Reply:
Problem je tady:
https://github.com/opendata-mvcr/ssp/blob/694c258303fcb73cd3096972fc097a4e0e5552eb/validation/src/main/java/cz/gov/ssp/SGoVIri.java#L23

Matchuje sa tu stale oproti `https://slovník.gov.cz`, zadna priloha ktera tady zacne `http://onto.fel.cvut.cz/ontologies/slovník/agendový/popis-dat` neprojde. Pro workaround postaci, kdyz zmeni pattern aby akceptoval zaroven i `http://onto.fel.cvut.cz/ontologies/slovník/agendový/popis-dat` no idealni reseni to neni.

Reply:
pořešíme dneska na meetingu